src/commands/help-command.js

```diff
@@ -0,0 +1,33 @@
+import { MessageEmbed } from 'discord.js';
+
+export class HelpCommand {
+    static getCommand() {
+        return 'help';
```

Comment:
Every command should have a static method called getCommand which should return the command string.